Resources/Prototypes/Entities/Objects/Specific/chemistry-vials.yml

```diff
@@ -149,7 +150,7 @@
         reagents:
           - ReagentId: Chlorine
             Quantity: 5
-            
+
```

Comment:
untouch

Reply:
`.editorconfig` setting (`trim_trailing_whitespace = true`) removed this on save. Added it back.